src/Response/Clients/ClientResponse.php

```diff
@@ -74,4 +76,23 @@ class ClientResponse extends BaseResponse
             'last_login',
             'signup_date',
         ];
+
+    /**
+     * LatePaymentReminders
+     *
+     * @return LatePaymentReminder[]|null
+     */
+    public function getLateRemindersAttribute(): ?array
```

Comment:
Add test that get the late reminder from client response.

Reply:
Don't forget to put the "include" argument in the test has this data is not given directly.